{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/taskapp/celery.py

```diff
@@ -75,7 +75,7 @@ def ready(self):
 
 @app.task(bind=True)
 def debug_task(self):
-    print('Request: {0!r}'.format(self.request))  # pragma: no cover
+    print(f'Request: {self.request!r}')  # pragma: no cover
```

Comment:
`!r` still works in f-string
See: https://www.python.org/dev/peps/pep-3101/#explicit-conversion-flag